frontend/js/ci.js

```diff
@@ -136,9 +232,78 @@ const displayGraph = (runs) => {
     window.onresize = function () { // set callback when ever the user changes the viewport
         chart_instance.resize();
     }
+
+    return chart_instance;
+}
+
+const displayAveragesTable = (runs) => {
+    let labels = new Set()
+    runs.forEach(run => {
+        let [value, unit, run_id, timestamp, label, cpu, commit_hash] = run;
+        labels.add(label)
+    });
```

Comment:
because run has no value label, it would be 

```
runs.forEach(run => {
    labels.add(run[0]);
  });
```
instead... which is probably still better than assigning a bunch of variables in a loop that never get used. I did it originally for readability and I think I thought at the time I would do more in that loop that required the extra variables. I condensed it now

Reply:
It still looks like the old form?

Reply:
True - not sure why my earlier change didn't get pushed but here it is now. (also, its run[4] for label, not run[0])